Prebuild script fails on Windows

Both building and testing fail on Windows due to the presence of a prebuild script including "rm -rf".

Windows does not necessarily have rm available, and if rm is not available, the prebuild script will fail and halt the build/run task.

It's not really a blocking issue for me since it doesn't look too difficult to workaround, but something to be aware of.

```
> Executing task: npm run test <


> kalidokit@1.1.2 test C:\Projects\Programming\kalidokit
> npm run build && npm run dev


> kalidokit@1.1.2 prebuild C:\Projects\Programming\kalidokit
> rm -rf dist lib

'rm' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! kalidokit@1.1.2 prebuild: `rm -rf dist lib`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the kalidokit@1.1.2 prebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
     },
     "devDependencies": {
         "prettier": "^2.4.1",
+        "rimraf": "^3.0.2",
         "rollup-plugin-friendly-type-imports": "^1.0.1",
         "typescript": "^4.4.4",
         "vite": "2.6.0",

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     ],
     "scripts": {
         "dev": "vite docs",
-        "prebuild": "rimraf dist lib",
+        "prebuild": "rimraf dist",
         "build": "vite build && npm run build:lib",
         "build:lib": "tsc -p .",
         "serve": "vite",

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     ],
     "scripts": {
         "dev": "vite docs",
-        "prebuild": "rm -rf dist lib",
+        "prebuild": "rimraf dist lib",
         "build": "vite build && npm run build:lib",
         "build:lib": "tsc -p .",
         "serve": "vite",

